core/modules/core/src/com/haulmont/yarg/formatters/impl/AbstractFormatter.java

```diff
@@ -120,7 +118,10 @@ protected String formatValue(Object value, String parameterName, String fullPara
         String valueString;
         String formatString = getFormatString(parameterName, fullParameterName);
         if (formatString != null) {
-            if (formatString.startsWith("class:")) {
+            if (Boolean.TRUE.equals(isGroovyScript(parameterName, fullParameterName))) {
+                DefaultScriptingImpl defaultScripting = new DefaultScriptingImpl();
```

Comment:
Work with `com.haulmont.yarg.util.groovy.Scripting` interface instead of implementations. Add separate property for scripting